mirrorlist/mirrorlist_statistics.py

```diff
@@ -21,18 +21,18 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
 
-import sys, pylab, time, getopt
+import sys
+import pylab
+import time
+import getopt
 
 start = time.clock()
 
 logfile = None
 dest = None
 offset = 0
+embargoed_countries = ['CU', 'IR', 'KP', 'SD', 'SY']
```

Comment:
Yes, changing the code for a configuration seems wrong. I will move it to the configuration file.

And yes, the DB has a table of embargoed countries, which is also used for the maps. It is only used by the map script so far, maybe it could also be removed from the DB and live in the configuration file only.